### ISSUE TITLE
Support i

### ISSUE THREAD

**Issue by iop883684** [facebook/FBNotifications#15]
- `FBNotifications` required by `Podfile`

Specs satisfying the `FBNotifications` dependency were found, but they required a higher minimum deployment target.

**Comment by nlutsenko** [comment]
Indeed, our current minimum supported deployment target is iOS 8.0.
We are planning to add iOS 7 support very soon, but in the meantime, please try to use the code from source and please comment here if you run into any issues.
